gulpfile.js

```diff
@@ -202,33 +236,41 @@ gulp.task('iconfont-svg-outline', function (cb) {
 		if (fs.existsSync('./.build/iconfont-unicode.json')) {
 			iconfontUnicode = require('./.build/iconfont-unicode');
 		}
-
+		
 		await asyncForEach(files, async function (file) {
-			const name = path.basename(file, '.svg'),
+			
+			const name = path.basename(file, '.svg');
+			
+			if (compileOptions.includeIcons.length==0 || compileOptions.includeIcons.indexOf(name)>=0) {
+					
 				unicode = iconfontUnicode[name];
 
-			await console.log('Stroke for:', file, unicode);
-
-			let strokedSVG = fs.readFileSync(file).toString();
-
-			strokedSVG = strokedSVG
-				.replace('width="24"', 'width="1000"')
-				.replace('height="24"', 'height="1000"');
-
-			await outlineStroke(strokedSVG, {
-				optCurve: false,
-				steps: 4,
-				round: 0,
-				centerHorizontally: true,
-				fixedWidth: true,
-				color: 'black'
-			}).then(outlined => {
-				if (unicode) {
-					fs.writeFileSync(`icons-outlined/u${unicode.toUpperCase()}-${name}.svg`, outlined);
-				} else {
-					fs.writeFileSync(`icons-outlined/${name}.svg`, outlined);
-				}
-			}).catch(error => console.log(error));
+				await console.log('Stroke for:', file, unicode);
+
+				let strokedSVG = fs.readFileSync(file).toString();
+
+				strokedSVG = strokedSVG
+					.replace('width="24"', 'width="1000"')
+					.replace('height="24"', 'height="1000"');
+					if (compileOptions.strokeWidth)
```

Comment:
Pls fix the identation here and add brackets - I know this if now works with the single statement below but it's a bit confusing to see and it's easy to forget about :D